core/src/main/java/edu/wpi/grip/core/operations/composite/DistanceTransformOperation.java

```diff
@@ -0,0 +1,124 @@
+
+package edu.wpi.grip.core.operations.composite;
+
+import com.google.common.eventbus.EventBus;
+
+import edu.wpi.grip.core.InputSocket;
+import edu.wpi.grip.core.Operation;
+import edu.wpi.grip.core.OutputSocket;
+import edu.wpi.grip.core.SocketHint;
+import edu.wpi.grip.core.SocketHints;
+
+import java.io.InputStream;
+import java.util.Optional;
+
+import org.bytedeco.javacpp.opencv_core.Mat;
+import static org.bytedeco.javacpp.opencv_core.*;
+import static org.bytedeco.javacpp.opencv_imgproc.*;
+
+/**
+ * GRIP {@link Operation} for
+ * {@link org.bytedeco.javacpp.opencv_imgproc#distanceTransform}.
+ *
+ * @author Sam Carlberg
+ */
+public class DistanceTransformOperation implements Operation {
```

Comment:
Distance transform sets the value of each pixel to its distance from the nearest black pixel, e.g.

![image](https://cloud.githubusercontent.com/assets/6320992/13857792/d6effe1a-ec51-11e5-9ab7-23b18bb335ae.png)

becomes

![image](https://cloud.githubusercontent.com/assets/6320992/13857816/e0dc32b8-ec51-11e5-9ea4-2a17fe46ed3b.png)

[Here's an example for distance transformation, normalization, and watershed](http://docs.opencv.org/3.1.0/d2/dbd/tutorial_distance_transform.html)